### PR TITLE
s3cmd image

### PR DESCRIPTION
**Pull request by piotr-putit**
- build image which then will be used to push contracts to s3
- drone s3 plugin breaks out connectivity during upload

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -90,6 +90,34 @@ pipeline:
...
+    image: docker:stable
+    commands:
+      - apk add git bash
+      - ./update-images.sh s3cmd-image
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    when:
+      changeset:
+        includes:
+          - "**./docker/s3cmd-image/*"
```
Comment (sacherjj) [inline .drone.yml:103]:
```suggestion
          - "**./s3cmd-image/*"
```

File: .drone.yml [hunk 2]
```diff
@@ -90,6 +90,34 @@ pipeline:
...
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    secrets:
+      - docker_username
+      - docker_password
+    when:
+      event: [ push, tag ]
+      changeset:
+        includes:
+          - "**./docker/s3cmd-image/*"
```
Comment (sacherjj) [inline .drone.yml:119]:
```suggestion
          - "**./s3cmd-image/*"
```

**Comment by sacherjj** [comment]
I removed the docker directory from all changesets in .drone.yml. This was a mistake on my part with the previous update.

**Review by sacherjj — APPROVED** [review]
With the .drone.yml changes I made, this looks good.

**Comment by piotr-putit** [comment]
bors try

**Pull request merged**